metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -23,6 +23,7 @@
 from sqlalchemy.exc import ProgrammingError
 from sqlalchemy.sql import sqltypes as types
 from sqlalchemy.types import TypeDecorator, TypeEngine
+from sqlalchemy_bigquery import STRUCT
```

Comment:
looks like this import is causing issues - this file is used by all ingestion sources, so it shouldn't have a hard requirement on bigquery in particular.

you can use `register_custom_type` in the athena.py or bigquery instead if that's needed

Reply:
Thanks for pointing out!

I'll have a look the upcoming days

Reply:
Should be resolved now 🙂